src/containers/AddProjectDetails/AddProjectDetailsForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box } from "@mui/material";
+import React from "react";
+import { useFormContext } from "react-hook-form";
+import { InputText } from "../../components/InputText/InputText";
+import * as styledComp from "./AddProjectDetailsForm.style";
+// import MUITextField from "../../mui-components/MUITextField/MUITextField";
+
+export default function AddProjectDetailsForm() {
+    // const {
+    //     register,
+    //     handleSubmit,
+    //     control,
+    //     reset,
+    //     setValue,
+    //     getValues,
+    //     setError,
+    //     formState: { errors },
+    //     formState,
+    // } = useFormContext();
```

Comment:
Can we get rid of the commented code if it's not required?

Reply:
sure